vendor/extensions/stories/app/views/refinery/stories/stories/index.html.erb

```diff
@@ -1,30 +1,42 @@
 <div class="StoriesPage">
   <div class="banner">
-    <div class="container">
-      <div class="content-column banner-content">
-        <h1>Weigh In</h1>
+    <div class="container"></div>
```

Comment:
Right now this container class is unused and technically could be removed, but I would suggest maybe following the pattern we use on the home page. See https://github.com/MAPC/digitalhub/blob/develop/app/views/refinery/pages/home.html.erb#L14-L39 for that example.